notebooks/community/pipelines/google_cloud_pipeline_template.ipynb

```diff
@@ -0,0 +1,1058 @@
+{
```

Comment:
<p>Where is the component?</p><p> </p><p>Why not directly call the predefined ModelUploadOp component?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>